loom/engine/sound/lsSound.cpp

```diff
@@ -116,6 +116,11 @@ class OALBufferNote
         buffer = 0;
         refCounter = 1;
     }
+    
+    ~OALBufferNote()
+    {
+        delete [] asset;
```

Comment:
Whoops, messed up the mebers. It's actually allocated by strdup, so it's fine by itself. But maybe we should replace strup and allocated memory with lmNew?